### PR TITLE
For #10406 - Expose Flank results as public artifacts

### PR DESCRIPTION
**Pull request by AaronMT**
For #10406

### CODE REVIEW

File: automation/taskcluster/androidTest/ui-test.sh [hunk 1]
```diff
@@ -118,20 +120,20 @@ function failure_check() {
...
     echo "COPY ARTIFACTS"
     echo
-    cp -r ./results /builds/worker/artifacts
+    #cp -r ./results /builds/worker/artifacts
 
     echo
     echo "RESULTS"
     echo
-    ls -la ./results
+    ls -la "${RESULT_DIR}"
```
Comment (rpappalax) [inline automation/taskcluster/androidTest/ui-test.sh:128]: should be `RESULTS_DIR` (plural)

File: automation/taskcluster/androidTest/ui-test.sh [hunk 2]
```diff
@@ -118,20 +120,20 @@ function failure_check() {
     echo
     echo "COPY ARTIFACTS"
     echo
-    cp -r ./results /builds/worker/artifacts
+    #cp -r ./results /builds/worker/artifacts
```
Comment (rpappalax) [inline automation/taskcluster/androidTest/ui-test.sh:123]: if this is not necessary, let's remove

**Review by rpappalax — APPROVED** [review]
LGTM

File: automation/taskcluster/androidTest/ui-test.sh (hunk 2 above)
Comment (AaronMT) [reply automation/taskcluster/androidTest/ui-test.sh:123]: 👍🏼

**Pull request merged**